### ISSUE TITLE
It keeps say 'Cannot find git-hooks. Did you install it?'

### ISSUE THREAD

**Issue by zjhiphop** [tarmolov/git-hooks-js#6]
I just followed the turtorial to test, but it fails, and keep said 'Cannot find git-hooks. Did you install it?'.

My env is Windows 7 + node 0.12.7

Thanks.

**Comment by theres** [comment]
hooks on windows looks try to use such require:
```javascript
#!/usr/bin/env node

try {
    require('..\..\node_modules\git-hooks\lib/git-hooks').run(__filename, process.argv[2], function (code) {
        process.exit(code);
    });
} catch (e) {
    console.error('Cannot find git-hooks. Did you install it?');
}
```
i.e. path separators are windows-type, require need *nix type separators. I'll try to prepare pull request today

**Comment by tarmolov** [comment]
Sorry for the silence, guys. I've being on vacation.

As you've already noticed I didn't test `git-hooks` on windows. That's because we use *nix and macs at work. 

It seems that it's time to make `git-hook` works on windows, too. I'll really any help to get it done.

**Comment by Andre-487** [comment]
I have the same error on MaxOS X.

I change hook file that way:
```js
#!/usr/bin/env node
try {
    require('../../node_modules/git-hooks/lib/git-hooks').run(__filename, process.argv[2], function (code) {
        process.exit(code);
    });
} catch (e) {
    console.error('Cannot find git-hooks. Did you install it?');
    throw e;
}
```

And get the true error:
```
Cannot find git-hooks. Did you install it?
/Users/andre487/IdeaProjects/opensearch-validator/.git/hooks/pre-commit:9
    throw e;
          ^
Error: spawn EACCES
    at exports._errnoException (util.js:746:11)
    at ChildProcess.spawn (child_process.js:1162:11)
    at exports.spawn (child_process.js:995:9)
    at runHooks (/Users/andre487/IdeaProjects/opensearch-validator/node_modules/git-hooks/lib/git-hooks.js:128:16)
    at Object.module.exports.run (/Users/andre487/IdeaProjects/opensearch-validator/node_modules/git-hooks/lib/git-hooks.js:108:13)
    at Object.<anonymous> (/Users/andre487/IdeaProjects/opensearch-validator/.git/hooks/pre-commit:4:59)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

Node.js 0.12.7
MaxOS X 10.10.5

**Comment by tarmolov** [comment]
Looks like a permission error. Could you check your pre-commit permissions?

**Comment by Andre-487** [comment]
Unfortunately I can't already check that version but I throw away Shell.js and replace it with Bash and now it works.

Permissions was correct I think because I launched this script dicectly from command line

**Comment by tarmolov** [comment]
Discussion is continued in https://github.com/tarmolov/git-hooks-js/issues/10

**Comment by codebling** [comment]
Should be fixed by #29
